packages/server/src/utils/ldapLogin.ts

```diff
@@ -3,6 +3,7 @@ import { User } from '../services/database/types';
 import Logger from '@joplin/utils/Logger';
 import { LdapConfig } from './types';
 import { ErrorForbidden } from './errors';
+import { readFileSync } from 'node:fs';
```

Comment:
Please use `readFile` from `fs/promises`. We generally don't use sync function since they block the main thread